<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CruvePlotWidget</class>
 <widget class="QWidget" name="CruvePlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CruvePlotWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3" rowstretch="100,5">
     <item row="0" column="0">
      <widget class="QCustomPlot" name="qCustomPlot" native="true"/>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout" columnstretch="20,10,40,10,20,0">
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 0);</string>
         </property>
         <property name="text">
          <string>纵坐标Acceleration(m/s²)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 170, 0);</string>
           </property>
           <property name="text">
            <string>Y轴加速度</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="radioButton_3">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 170, 0);</string>
           </property>
           <property name="text">
            <string>Z轴加速度</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 0, 127);</string>
           </property>
           <property name="text">
            <string>X轴加速度</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 0);</string>
         </property>
         <property name="text">
          <string>横坐标时间刻度(ms)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="SetButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(120, 31, 197);</string>
         </property>
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
